src/Application/Drive/View/Sidebar.elm

```diff
@@ -523,3 +560,34 @@ detailsForSelection { showPreviewOverlay } sidebar model =
             |> Maybe.withDefault
                 nothing
         ]
+
+
+
+-- UTILITIES
+
+
+onCtrlS : msg -> Html.Attribute msg
+onCtrlS message =
+    let
+        ensureEquals value decoder =
+            decoder
+                |> D.andThen
```

Comment:
Not a necessary change, but when there's only a single pipe, feel free to remove it.
```
Decode.andThen
  (\val -> ...)
  decoder
```

Reply:
I just think it reads a little nicer with `andThen` being between its two arguments. "Decoder and then, if ..."

Reply:
Good point 👍